lib/collection/src/operations/types.rs

```diff
@@ -385,13 +387,13 @@ impl From<tonic::Status> for CollectionError {
     fn from(err: tonic::Status) -> Self {
         match err.code() {
             tonic::Code::InvalidArgument => CollectionError::BadInput {
-                description: "InvalidArgument".to_string(),
+                description: format!("InvalidArgument: {}", err),
```

Comment:
If you'd like, you can change them like this:
```suggestion
                description: format!("InvalidArgument: {err}"),
```

Reply:
let's maybe do it some in some other pr, cause there are currently all other code use the first version